app/forms.py

```diff
@@ -0,0 +1,60 @@
+"""
+Forms for registering doctors, patients, and prescriptions.
+"""
+
+from flask_wtf import FlaskForm
+from flask_wtf.file import FileRequired
+from wtforms import FileField, PasswordField, StringField, SubmitField, IntegerField
+from wtforms.validators import DataRequired
+
+
+class InfoForm(FlaskForm):
+    """Base class for registering names and email."""
+
+    firstname = StringField("First name", validators=[DataRequired()])
+    lastname = StringField("Last name", validators=[DataRequired()])
+    email = StringField("Email", validators=[DataRequired()])
+
+
+class UserForm(FlaskForm):
+    """Base class for registering a username and password."""
+
+    username = StringField("Username", validators=[DataRequired()])
+    password = PasswordField("Password", validators=[DataRequired()])
+
+
+class RegistrationForm(InfoForm, UserForm):
+    """Final class for registering new users (doctors)."""
+
+    submit = SubmitField("Submit")
+
+
+class LoginForm(UserForm):
+    """Final class for logging in existing users (doctors)."""
+
+    subimt = SubmitField("Login")
```

Comment:
Typo `submit`

Reply:
lol thnx for this! i must not have rebuilt the image when i tested it, because otherwise it wouldn't have worked